connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/GrpcStub.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.connect.client
+
+import io.grpc.ManagedChannel
+
+import org.apache.spark.connect.proto.{AnalyzePlanRequest, AnalyzePlanResponse, ConfigRequest, ConfigResponse, ExecutePlanRequest, ExecutePlanResponse, InterruptRequest, InterruptResponse}
+import org.apache.spark.connect.proto
+
+private[client] class GrpcStub(channel: ManagedChannel) {
```

Comment:
This should extend `SparkConnectServiceBlockingStub` instead

Reply:
Except the class is final ...

Reply:
Yes, I once thought about using `SparkConnectServiceBlockingStub` but it turns out the class is final...

Reply:
@grundprinzip was your concern that without extending that stub there are no guarantees that this class will adhere to the interface?
But on the other hand, if it was extending the stub, and forgot about overriding one of the RPC methods, it could go unnoticed that it falls through to super? As it is now, the missing method will be noticed.

Reply:
Correct, i was looking for a way to leave a trace of the contract of the interface.

I spent quite some time reading through the different alternatives and it's fine with me now.